setup.py

```diff
@@ -78,7 +78,7 @@ def get_extras_require() -> Dict[str, List[str]]:
             "nbval",
             "pytorch-ignite",
             "scikit-image",
-            "scikit-learn",
+            "scikit-learn>=0.19.0,<0.23.0",  # optuna/visualization/param_importances.py.
```

Comment:
Just for the record, importance-releated features should work with newer e.g. 0.23.1 as well.